src/main/webapp/js/maps-page-loader.js

```diff
@@ -7,6 +7,11 @@ const MAP_NIGHT_TYPE_ID = 'night'
  * New modes/map styles can be added in extraMapStyles.json file.
  *
  */
+
+let map;
+/* Editable marker that displays when a user clicks in the map. */
```

Comment:
The comment is a little unclear to me.. do you mean a set marker is made visible as soon as the user clicks on the map, or do you mean the marker is used to show the instance/ time when a users clicks on the map?

Reply:
It means the variable `editMarker` is used to contain the editable marker that shows up when a user clicks in the map